browser/BUILD.gn

```diff
@@ -246,6 +247,7 @@ source_set("browser_process") {
 
     deps += [
       "//brave/components/binance/browser",
+      "//brave/components/ntp_widget_utils/browser",
```

Comment:
are these needed here? Seems like it's only used by the api?

Reply:
It's used by `BinanceService` as well: https://github.com/brave/brave-core/blob/master/components/binance/browser/binance_service.cc#L24

Reply:
nope, not needed here, removed